template/en/default/robots.txt.tmpl

```diff
@@ -2,7 +2,8 @@ User-agent: *
 Disallow: [% basepath FILTER none %]
 Crawl-delay: 30
 
-[% IF NOT Bugzilla.localconfig.urlbase.matches("bugzilla-dev") %]
+[% USE Bugzilla %]
+[% IF NOT Bugzilla.localconfig.urlbase.match("bugzilla-dev") %]
```

Comment:
Lets make is so we only generate a permissive robots.txt on production, rather than just excluding bugzilla-dev.  That'll make sure that any transient/test instances we spin up in future won't be indexed.